src/utility.h

```diff
@@ -49,6 +49,12 @@ void strip(std::string &s, UNARY f = chkspace) {
     rstrip(s, f);
 }
 
+template <typename UNARY = bool(char)>
+std::string stripped(std::string s, UNARY f = chkspace) {
```

Comment:
Since I'm calling `strip()` here I need to use the same template declaration it has. I guess this could also be done without a template these days. By using the template this could also be used with a different function signature that somehow evalutes to bool. Although I don't believe this feature is used in the codebase currently.